admin/index_dashboard.php

```diff
@@ -39,41 +65,30 @@
     ?>
 
     <div id="colone" class="col-xs-12 col-sm-6 col-md-4 col-lg-4">
-
-        <?php
-        include DIR_WS_MODULES . 'dashboard_widgets/BaseStatisticsDashboardWidget.php';
-        ?>
-
-        <?php
-        include DIR_WS_MODULES . 'dashboard_widgets/SpecialsDashboardWidget.php';
-        ?>
-
-        <?php
-        include DIR_WS_MODULES . 'dashboard_widgets/OrderStatusDashboardWidget.php';
-        ?>
-
+    <?php
+    foreach ($widgets as $widget) {
+        if ($widget['column'] === 1) {
+            include $widget['path'];
```

Comment:
Why `include` instead of `require`?

Reply:
Mainly just because that's what it was already using.

I tend to use `include` when I don't care if the loading of the file fails. Here I'm content to let the script continue even if the file fails to `include`; whereas calling `require` would emit a hard failure error and halt execution.